seq2science/scripts/scRNA-seq-1.0.0-rc.1/analysis/kb_seurat_pp.rmd

```diff
@@ -0,0 +1,692 @@
+---
+title: "SORT-seq dataset with velocity preparations"
+params:
+  method: 
+    label: "method -- experimental method"
+    choices: ['384plate', 'droplet']
+    value: '384plate'
+  kb.dir: 
+    label: "kb.dir -- directory containing kb-python output"
+    value: /home/user/experiment/results/kallistobus/
```

Comment:
are these default values?

Reply:
These are example values which will be overwritten when knitting the document. I can remove them so it is a bit clearer.

Reply:
if they are just examples its fine! I don't know as much about Rmds as I'd want to (yet)

Reply:
Removed hard-coded paths